packs/bestiary-family-ability-glossary/protean-warpwave.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>Many proteans can subject their foes to disorienting alterations perceived in time and space by creating ripples of unstable reality in the environment called warpwaves. When a creature fails its saving throw and is affected by a warpwave, roll [[/gmr 1d8 #Warpwave]]{1d8} and consult the table below for the specific effect on that creature. Unless indicated otherwise, a warpwave effect lasts for [[/gmr 1d4 #Duration Warpwave Effect]]{1d4 rounds}, and a new warpwave effect negates any previous warpwave effect already affecting a creature.</p><table class=\"pf2e remaster\"><thead><tr><th>[[/gmr 1d8 #Warpwave]]{1d8}</th><th>Warpwave Effect</th></tr></thead><tbody><tr><td>1</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 2} (@UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 3} on a critical failure)</td></tr><tr><td>2</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Confused] and gains [[/r 4d6 #Temporary Hit Points]]{4d6 temporary Hit Points}</td></tr><tr><td>3</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Dazzled] (permanent on a critical failure)</td></tr><tr><td>4</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 2} (@UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 3} on a critical failure)</td></tr><tr><td>5</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Immobilized] by filaments of energy</td></tr><tr><td>6</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Quickened] (Stride, Strike, or Step only)</td></tr><tr><td>7</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Slowed]{Slowed 1}</td></tr><tr><td>8</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 2} (@UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 3} on a critical failure)</td></tr></tbody></table>"
```

Comment:
```suggestion
            "value": "<p>Many proteans can subject their foes to disorienting alterations perceived in time and space by creating ripples of unstable reality in the environment called warpwaves. When a creature fails its saving throw and is affected by a warpwave, roll [[/gmr 1d8 #Warpwave]] and consult the table below for the specific effect on that creature. Unless indicated otherwise, a warpwave effect lasts for [[/gmr 1d4 #Duration Warpwave Effect]]{1d4 rounds}, and a new warpwave effect negates any previous warpwave effect already affecting a creature.</p><table class=\"pf2e remaster\"><thead><tr><th>[[/gmr 1d8 #Warpwave]]</th><th>Warpwave Effect</th></tr></thead><tbody><tr><td>1</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 2} (@UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 3} on a critical failure)</td></tr><tr><td>2</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Confused] and gains [[/r 4d6 #Temporary Hit Points]]{4d6 temporary Hit Points}</td></tr><tr><td>3</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Dazzled] (permanent on a critical failure)</td></tr><tr><td>4</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 2} (@UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 3} on a critical failure)</td></tr><tr><td>5</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Immobilized] by filaments of energy</td></tr><tr><td>6</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Quickened] (Stride, Strike, or Step only)</td></tr><tr><td>7</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Slowed]{Slowed 1}</td></tr><tr><td>8</td><td>@UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 2} (@UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 3} on a critical failure)</td></tr></tbody></table>"
```

Reply:
Probably don't need the d8 roll in both the description and the table header?